### PR TITLE
This is new commit Java

### PR DESCRIPTION
**Pull request by guppi4u**
Hi I have added  new file 
Please merge my pull request 

guppi4u

### CODE REVIEW

**Comment by guppikan** [comment]
please update your file correctly

File: newFile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+added content to the file
```
Comment (guppikan) [inline newFile.txt]: please write something meaningful

**Review by guppikan — CHANGES_REQUESTED** [review]
please change

**Review by guppi4u — COMMENTED** [review]
Good !!!

**Comment by guppi4u** [comment]
Closing issue

**Review by guppikan — COMMENTED** [review]
Done

**Pull request closed without merging**